samples/connectors/src/main/java/quarks/samples/connectors/iotf/IotfSensors.java

```diff
@@ -109,6 +99,31 @@ public static void simulatedSensors(IotDevice device, boolean print) {
         // with event identifier "sensors".
         device.events(sensors, "sensors", QoS.FIRE_AND_FORGET);
     }
+    
+    /**
+     * Create a heart beat device event with
+     * identifier {@code heartbeat} to
+     * ensure there is some immediate output and
+     * the connection to IoTF happens as soon as possible.
+     * @param device IoT device
+     */
+    public static void heartBeat(IotDevice device, boolean print) {
```

Comment:
I guess I'm saying that some applications may have a role for a heartbeat that cannot be implemented by KA/LWT, in that the heartbeat (not this implementation) can contain up to date information about a device.